cloudinit/config/cc_apt_configure.py

```diff
@@ -6,225 +6,364 @@
+                    'uniqueItems': True,
+                    'description': dedent("""\
+                        Entries in the sources list can be disabled using
+                         ``disable_suites``, which takes a list of suites
+                         to be disabled. If the string ``$RELEASE`` is
+                         present in a suite in the ``disable_suites`` list,
+                         it will be replaced with the release name. If a
+                         suite specified in ``disable_suites`` is not
+                         present in ``sources.list`` it will be ignored.
+                         For convenience, several aliases are provided for
+                         ``disable_suites``:
+
+                             ``updates`` => ``$RELEASE-updates``,
+                             ``backports`` => ``$RELEASE-backports``.
+                             ``security`` => ``$RELEASE-security``,
+                             ``proposed`` => ``$RELEASE-proposed``,
+                             ``release`` => ``$RELEASE``.
+
+                        When a suite is disabled using ``disable_suites``,
+                        its entry in ``sources.list`` is not deleted; it
+                        is just commented out.""")
+                },
+                'primary': {
+                    **mirror_property,
+                    'description': dedent("""\
+                        The primary and security archive mirrors can
+                        be specified using the ``primary`` and
+                         ``security`` keys, respectively Both the
+                         ``primary`` and ``security`` keys take a list
+                        of configs, allowing mirrors to be specified
+                        on a per-architecture basis. Each config is a
+                        dictionary which must have an entry for
+                         ``arches``, specifying which architectures
+                        that config entry is for. The keyword
+                         ``default`` applies to any architecture not
+                        explicitly listed. The mirror url can be specified
+                        with the ``uri`` key, or a list of mirrors to
+                        check can be provided in order, with the first
+                        mirror that can be resolved being selected. This
+                        allows the same configuration to be used in
+                        different environment, with different hosts used
+                        for a local apt mirror. If no mirror is provided
+                        by ``uri`` or ``search``, ``search_dns`` may be
+                        used to search for dns names in the format
+                         ``<distro>-mirror`` in each of the following:
+
+                            fqdn of this host per cloud metadata,
+                            localdomain,
+                            domains listed in ``/etc/resolv.conf``.
+
```

Comment:
Each of these lines minimally needs a leading hyphen for bullet points in sphinx (read the docs)
test with `tox -e doc` locally 
```suggestion
                           - fqdn of this host per cloud metadata,
                           - localdomain,
                           - domains listed in ``/etc/resolv.conf``.

```

Reply:
I have added the bullet points back, but due to problem on the description formatting, they will not be correctly displayed. But I will think in a solution for this problem